src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -31,7 +28,7 @@ const iconSrcs: Array<string> = Object.values(icons).map(src => String(src));
 type Props = {
   project: Project,
   isVisible: boolean,
-  queueEmpty: boolean,
+  dependenciesChangingForProject: boolean,
```

Comment:
I renamed this prop because I felt like the `ProjectConfigurationModal` doesn't need to know about the queue status (or even what a queue is), it just needs to know whether there's a blocking action in progress.

I imagine this will help new contributors (who haven't discovered the dependency queue system yet) understand what this condition is.

Reply:
I think we could also name this `savingNotBlocked` or `dependencyChangingDone` so it's clear that if true the button is enabled.

Reply:
Ah, right! I changed the logic so that the current variable name actually made sense, haha.

Let me know if I should revert it and use one of the names you suggested (I think it's slightly easier to understand if the "true" state is the blocked state, but I don't feel that strongly)

Reply:
I wouldn't invert the logic as I think it's good to match `isQueueEmpty` logic.
Yes, I think reverting would be great and take one of my suggestions.